docs/make.jl

```diff
@@ -5,49 +5,20 @@ using DimensionalData
 makedocs(; sitename="DimensionalData", authors="Rafael Schouten et al.",
```

Comment:
see suggestion in https://github.com/rafaqz/DimensionalData.jl/issues/648, it might fix the missing docs.